05-cluster-prerequisites.md

```diff
@@ -35,14 +35,25 @@ Following the steps below will result in the provisioning of the shared Azure re
+  >
+  > The app team is instructed to build the smallest container images they can. This is something they could achieve by following the [builder pattern](https://docs.docker.com/develop/develop-images/multistage-build/#before-multi-stage-builds) or the [mutli-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds), both approaches will produce final smaller container images that are meant to be for runtime-only. This will be beneficial in many ways but mainly in the speed of replication as well as in the transfer costs. A great feature as part of the `Geo-Replication` is that it will only replicates unique layers, also reducing data transfer across regions.
+  >
+  > In case of a region is down, the app team is now covered by the Azure Traffic Manager that comes on the scene to help deriving traffic to the registry located in the region that is closest to their multiple clusters in terms of network latency.
+  >
+  > After this initial design decision at the ACR level, the app team can also consider analyzing how they could tactically expand into [Availability Zones](https://docs.microsoft.com/en-us/azure/container-registry/zone-redundancy) as a way of being even more resilient.
+  >
+  > :bulb: Another benefit of having `Geo-Replication` is that security permissions are now centralized in a single registry instance. That said, if your organization shares the _User Identities_ among the container hosts, this simplifies the security management a lot. In this reference implementation, the design decision is not to share a single _User Identity_ among the multiple clusters, otherwise that single identity will be granted with permissions to access other Azure resources that are meant to be cluster-specific such us Azure Key Vault. As things develop, the combination of [Availability Zones](https://docs.microsoft.com/en-us/azure/container-registry/zone-redundancy), currently in _Preview_,  for redundancy within a region, and geo-replication across multiple regions, is the recommendation when looking for the highest reliability and performance of a registry.
```

Comment:
```suggestion
  > :bulb: Another benefit of having `Geo-Replication` is that security permissions are now centralized in a single registry instance. That said, if your organization shares the _User Identities_ among the container hosts, this simplifies the security management a lot. In this reference implementation, the design decision is not to share a single _User Identity_ among the multiple clusters, otherwise that single identity will be granted with permissions to access other Azure resources that are meant to be cluster-specific such us Azure Key Vault. As things develop, the combination of [Availability Zones](https://docs.microsoft.com/azure/container-registry/zone-redundancy), currently in _Preview_,  for redundancy within a region, and geo-replication across multiple regions, is the recommendation when looking for the highest reliability and performance of a container registry.
```